src/gui/CMakeLists.txt

```diff
@@ -173,7 +179,7 @@ set(TS_FILES
 )
 
 set_source_files_properties(${TS_FILES} PROPERTIES OUTPUT_LOCATION "${CMAKE_CURRENT_SOURCE_DIR}/res/lang")
-if (Qt5_FOUND)
+if(Qt5_FOUND)
```

Comment:
I don't understand why this was changed. Ideally, all changes should follow the existing coding style - including in CMake. In this case, we should keep a single space.

Reply:
typo, fixed now